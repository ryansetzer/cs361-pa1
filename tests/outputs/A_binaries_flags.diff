--- outputs/A_binaries_flags.txt	2022-09-28 16:08:43.451620063 -0400
+++ expected/A_binaries_flags-2.txt	2022-09-11 14:59:21.000000000 -0400
@@ -1,6 +1,19 @@
 $ ./bin/ls -a data
+.
+..
+pwd.txt
+.hidden.txt
+empty.txt
 $ ./bin/ls -s data
+9 pwd.txt
+0 empty.txt
 $ ./bin/ls -sa data
+4096 .
+4096 ..
+9 pwd.txt
+5 .hidden.txt
+0 empty.txt
 $ ./bin/head -n 1 Makefile
+#
 $ quit
 
