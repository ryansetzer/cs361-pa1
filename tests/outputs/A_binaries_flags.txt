$ ./bin/ls -a data
.
..
pwd.txt
.hidden.txt
empty.txt
$ ./bin/ls -s data
9 pwd.txt
0 empty.txt
$ ./bin/ls -sa data
4096 .
4096 ..
9 pwd.txt
5 .hidden.txt
0 empty.txt
$ ./bin/head -n 1 Makefile
#
$ quit

