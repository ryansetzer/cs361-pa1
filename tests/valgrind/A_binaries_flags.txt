==59435== Memcheck, a memory error detector
==59435== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==59435== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==59435== Command: ../dukesh -b scripts/bins_flags.txt
==59435== 
==59435== 
==59435== HEAP SUMMARY:
==59435==     in use at exit: 0 bytes in 0 blocks
==59435==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==59435== 
==59435== All heap blocks were freed -- no leaks are possible
==59435== 
==59435== For lists of detected and suppressed errors, rerun with: -s
==59435== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
