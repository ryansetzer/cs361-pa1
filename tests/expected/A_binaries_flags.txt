$ ./bin/ls -a data
.
..
.hidden.txt
empty.txt
pwd.txt
$ ./bin/ls -s data
0 empty.txt
9 pwd.txt
$ ./bin/ls -sa data
4096 .
4096 ..
5 .hidden.txt
0 empty.txt
9 pwd.txt
$ ./bin/head -n 1 Makefile
#
$ quit

